### ISSUE TITLE
texbox2 error in MacOS High Sierra

### ISSUE THREAD

**Issue by iPsych** [psychopy/psychopy#3326]
Below error happens when put the textbox in experiment via builder version 2020.2.
Tested in simple experiment with only the textbox,and one text with textbox.
```
7.0344     ERROR     Unknown colorSpace: named
2020-11-27 19:33:28.494 python[37256:90572] ApplePersistenceIgnoreState: Existing state will not be touched. New state will be written to (null)
Traceback (most recent call last):
  File "/Users/iPsych/Documents/ee_lastrun.py", line 110, in <module>
    autoLog=True,
  File "/Applications/PsychoPy.app/Contents/Resources/lib/python3.6/psychopy/visual/textbox2/textbox2.py", line 211, in __init__
    self.text = text if text is not None else ""
  File "/Applications/PsychoPy.app/Contents/Resources/lib/python3.6/psychopy/tools/attributetools.py", line 32, in __set__
    newValue = self.func(obj, value)
  File "/Applications/PsychoPy.app/Contents/Resources/lib/python3.6/psychopy/visual/textbox2/textbox2.py", line 293, in text
    self._layout()
  File "/Applications/PsychoPy.app/Contents/Resources/lib/python3.6/psychopy/visual/textbox2/textbox2.py", line 469, in _layout
    self.glFont.upload()
  File "/Applications/PsychoPy.app/Contents/Resources/lib/python3.6/psychopy/visual/textbox2/fontmanager.py", line 504, in upload
    self.atlas.upload()
  File "/Applications/PsychoPy.app/Contents/Resources/lib/python3.6/psychopy/visual/textbox2/fontmanager.py", line 301, in upload
    gl.GL_ALPHA, gl.GL_UNSIGNED_BYTE, self.data)
ctypes.ArgumentError: argument 9: <class 'TypeError'>: wrong type
```

**Comment by peircej** [comment]
yes, apologies for that. The issue is fixed in 1e4e9f9cb40d0a4d931ac503d3dce4818d0afe8a and that's being baked into a release (2020.2.9) today

**Comment by iPsych** [comment]
Now it can save the unicode text input properly, but NOT shows any text in textinput2 box contents even english one with Arial font setting.
![Screen Shot 2020-11-27 at 11 56 45 PM](https://user-images.githubusercontent.com/28276515/100461619-63e12480-310c-11eb-8c29-f5c3bff1bfed.png)
